### ISSUE TITLE
Preview composables should not be public doesn't work for @preview function without @PreviewParameter

### ISSUE THREAD

**Issue by pokk** [twitter/compose-rules#119]
**Describe the bug**
According to [Preview composables should not be public](https://twitter.github.io/compose-rules/rules/#preview-composables-should-not-be-public), a preview function should give us a lint error/warning. However, it doesn't.

**To Reproduce**
```kotlin
@Preview
@Composable
fun Preview() {
    MaterialTheme {
        TestScreen()
    }
}
```

**Expected behavior**
It should have a lint error/warning for a public preview function.

**Screenshots**
<img width="346" alt="2023-05-03-3-59-17" src="https://user-images.githubusercontent.com/5198104/235830402-62aa6bac-373e-4a26-a902-3b899c642c27.png">

<img width="722" alt="2023-05-03-4-01-17" src="https://user-images.githubusercontent.com/5198104/235830582-76ced1d4-446a-45bf-b512-3028282d6415.png">

**Environment**
lib version: `detekt-twitter-compose-0.0.26-all.jar`
Android Studio: Android Studio Giraffe | 2022.3.1 Beta 1
OS: macOs Ventura 13.3.1

**Additional context**
Only a preview function with `@PreviewParameter` will have a lint error/warning.

https://github.com/twitter/compose-rules/blob/main/rules/common/src/main/kotlin/com/twitter/compose/rules/ComposePreviewPublic.kt#L22-L28
Just guessing, it might have the intention to do this. According to [Preview composables should not be public](https://twitter.github.io/compose-rules/rules/#preview-composables-should-not-be-public), it doesn't describe clearly. Either one might be wrong 🤔

**Comment by mtkw0127** [comment]
I'm also facing the same problem.
I set previewPublicOnlyIfParams false.
https://twitter.github.io/compose-rules/detekt/
related Issue: https://github.com/twitter/compose-rules/issues/95
```
  PreviewPublic:
    active: true
    # You can optionally disable that only previews with @PreviewParameter are flagged
    # previewPublicOnlyIfParams: false
```

**Comment by mtkw0127** [comment]
Discussing here.
https://github.com/mrmans0n/compose-rules/issues/24

**Comment by pokk** [comment]
Will close since there is the same discussion
